Makefile

```diff
@@ -222,6 +228,26 @@ else
 	cd devworkspace-crds && git checkout $(DEVWORKSPACE_API_VERSION) && git reset --hard origin/$(DEVWORKSPACE_API_VERSION)
 endif
 
+### update_terminal_manifests: pull latest web terminal manifests to web-terminal-operator. Note: pulls master branch by default
+update_terminal_manifests:
+	mkdir -p web-terminal-operator
+	cd web-terminal-operator && git init || true
+ifneq ($(shell git --git-dir=web-terminal-operator/.git remote), origin)
```

Comment:
This line reintroduces https://github.com/eclipse/che/issues/17180 -- `git --git-dir=web-terminal-operator/.git remote` is evaluated at startup and will fail if `./web-terminal-operator` does not exist.

Consider reworking this rule similarly to [#113](https://github.com/devfile/devworkspace-operator/pull/113/files#diff-b67911656ef5d18c4ae36cb6741b7965R206-R226)

Reply:
Oops, fixed on my other computer but forgot to push it from there haha . I'll fix it on my work laptop then push again